backend/src/main/java/cloudflight/integra/backend/model/Proposal.java

```diff
@@ -0,0 +1,46 @@
+package cloudflight.integra.backend.model;
+
+public class Proposal{
+    private Long id;
+    private String name;
+    private Type type;
+    private String status; //pending, approved, rejected
```

Comment:
Now that I think about it, the `status` would be better to be an Enum.
Then, we'd have no need for the comment

```java
public enum ProposalStatus {
    PENDING,
    APPROVED,
    REJECTED;
}
```